core/next.config.ts

```diff
@@ -38,7 +38,9 @@ async function writeSettingsToBuildConfig() {
     locales: data.site.settings?.locales,
     urls: {
       ...data.site.settings?.url,
-      cdnUrl: process.env.NEXT_PUBLIC_BIGCOMMERCE_CDN_HOSTNAME ?? data.site.settings?.url.cdnUrl,
+      cdnUrls: process.env.NEXT_PUBLIC_BIGCOMMERCE_CDN_HOSTNAME
+        ? process.env.NEXT_PUBLIC_BIGCOMMERCE_CDN_HOSTNAME.split(',').map((s) => s.trim())
+        : [data.site.settings?.url.cdnUrl],
```

Comment:
Should we filter out nullish values so we can guarantee this will be an array of strings?

Reply:
Added filtering out nullish values